apps/src/componentLibrary/checkbox/Checkbox.story.tsx

```diff
@@ -4,7 +4,7 @@ import {Meta, Story} from '@storybook/react';
 import {ComponentSizeXSToL} from '@cdo/apps/componentLibrary/common/types';
 
 export default {
-  title: 'DesignSystem/Checkbox',
+  title: 'DesignSystem/Checkbox', // eslint-disable-line storybook/no-title-property-in-meta
```

Comment:
Just confirming that you are assigning the title for 'DesignSystem' components so that this name is used instead of 'componentlLibrary'. Could we just rename the 'apps/src/componentLibrary' directory to 'apps/src/designComponents' and then not have to include this ignore rule each time we add a new design system component?

Reply:
I'm also curious about the order of the directories in storybook. I see 'DesignSystem' first, but then 'Templates'. I thought the order would be alphabetical since it is in apps.

Reply:
I added some configuration that allows us to force design system components (and the templates and code-studio directories after that) to the top of Storybook (I feel like this is desirable, at least the design system part, not sure if others agree):

https://github.com/code-dot-org/code-dot-org/pull/56200/files#diff-6547070ce8b34eb95836426236464d696f30d39efe116477c3bbb02c61d569c9

No strong opinions on whether its called "design system" or "component library"

Reply:
Any idea on why the listing of the non-configured directories are not in alpha order?

Reply:
I don't know 😬